packages/application-shell/src/components/navbar/navbar.mod.css

```diff
@@ -294,6 +341,21 @@
   display: none;
 }
 
+.sublist-new {
+  padding: var(--spacing-m) var(--spacing-l);
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
+  font-weight: var(--font-weight-for-navbar-link);
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
+  font-size: var(--font-size-for-navbar-link);
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
+  background-color: var(--background-color-for-navbar);
+  left: 58px;
+  z-index: -1;
+  list-style: none;
+  position: fixed;
+  display: none;
+}
+
```

Comment:
I think it's redundant (added 2 times in this file)

Reply:
Good catch! [33821ab](https://github.com/commercetools/merchant-center-application-kit/pull/3191/commits/33821ab1f3e635251915717f6846bf21137138fa)